pkg/jx/cmd/install.go

```diff
@@ -1179,3 +1194,55 @@ func (options *InstallOptions) dockerRegistryValue() (string, error) {
 	}
 	return "", nil
 }
+
+func (options *InstallOptions) getBinaryRepositoryManager() (error, string) {
+	binaryRepositoryManager := options.Flags.BinaryRepositoryManager
+	if binaryRepositoryManager != "" {
+		binaryRepositoryManager = strings.TrimSpace(binaryRepositoryManager)
+		return nil, strings.Title(binaryRepositoryManager)
+	}
+	// configure Nexus by default in headless mode
+	if options.Headless {
+		binaryRepositoryManager = NexusRepoMgr
+	}
+	if options.AdminSecretsService.Flags.ArtifactoryUrl != "" {
+		return nil, ArtifactoryRepoMgr
+	}
+	binaryRepositoryManagers := []string{NexusRepoMgr, ArtifactoryRepoMgr}
+	prompts := &survey.Select{
+		Message: "Select binary repository manager:",
+		Options: binaryRepositoryManagers,
+		Default: NexusRepoMgr,
+		Help:    "Binary repository manager to resolve dependencies and for uploading builds",
+	}
```

Comment:
We should probably make it clear that artifactory is assumed to already be running outside the cluster somewhere, ideally instead we'd optionally install and configure it incluster if folks want.  Maybe for now we just make it clear that Nexus is installed while an existing Artifactory is needed?

Reply:
Changed "Artifactory" --> "External Artifactory".